src/components/Icon/Icon.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import React from 'react'
+import Icons from 'react-native-vector-icons/FontAwesome5'
+
+import type { ViewStyleProp, } from 'react-native'
+import type { FontAwesome5Glyphs, } from 'react-native-vector-icons/FontAwesome5'
+
+type Props = {
+  name: FontAwesome5Glyphs,
+  size: number,
+  color: string,
+  style: ViewStyleProp
+}
+
+const Icon = ({ name, size = 33, color = '#e6e5e6', style, }: Props) => {
+  const IconComponent = Icons
+
+  return <IconComponent name={name} size={size} color={color} style={style} />
```

Comment:
Why not `<Icons ... />`?

Reply:
Because this component represents only one icon.